src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,5 @@
+$cell-size: 100px;
```

Comment:
Consider using a more descriptive name for the cell size, such as $day-cell-size.